lib/utils.js

```diff
@@ -62,11 +63,7 @@ exports.wetag = function wetag(body, encoding){
  * @api private
  */
 
-exports.isAbsolute = function(path){
-  if ('/' == path[0]) return true;
-  if (':' == path[1] && '\\' == path[2]) return true;
-  if ('\\\\' == path.substring(0, 2)) return true; // Microsoft Azure absolute path
-};
+exports.isAbsolute = pathIsAbsolute;
```

Comment:
Looking into this, it's unfortunate that since `path-is-absolute` is actually exporting a platform-specific absolute function, this check is not actually backwards-compatible. Yes, `path-is-absolute` is doing the "better thing", but I fear what this could break. The problem really comes down to Windows: if people are giving the path `'/some/layout/index.hbs'`, our views currently thinks it's absolute and we do not prepend the `root`, and so on Windows, it'll map to `'C:\\some\\layout\\index.hbs'`, where the drive is dependent on `process.cwd()`.

Thoughts?

We should definately move exclusively to this module in 5.x, but in 4.x, we can probably just do `return pathIsAbsolute.posix(path) || pathIsAbsolute.win32(path)`.